### ISSUE TITLE
Clone / Copy task with context

### ISSUE THREAD

**Issue by SamHasler** [choppedcode/vscode-tasks-and-contexts#7]
Sometimes I start working on something else and opening files without starting a new task. It would be helpful to be able to clone/copy the current task to a new task with the existing context information.

**Comment by kdescoteaux** [comment]
I agree.  This is something that I used to do with Mylyn all the time.  After cloning the task you can prune files from the contexts of each to clean them up.
